### ISSUE TITLE
SMT4 kill animations not displaying properly on Vulkan

### ISSUE THREAD

**Issue by leonmustdie** [Lime3DS/Lime-3DS-Emulator#82]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

2108

### Description of Issue

https://github.com/Lime3DS/Lime3DS/assets/166176656/f32ccc0c-72e7-4792-b2b6-a7c157569b0b
_Shin Megami Tensei 4_ displays graphics for when an enemy is killed with graphics pertaining to the type of move that was used (guns, ice, fire, ect). In-game, these are supposed to be overlays, but Lime instead displays it with a black box over the enemy.

### Expected Behavior

This is how the graphics are supposed to look. They overlay over the enemies, usually fading their sprites or erasing parts of them accordingly. This image was done using the Electric element. 

![firefox_W9Ei3Kpgxf](https://github.com/Lime3DS/Lime3DS/assets/166176656/0bb3c6cd-f936-4ea3-8f59-457d8b0715e1)

### Reproduction Steps

A simple way to reproduce the issue is to go into a battle and then kill an enemy to see the bugged kill animations. This can be done very early on. The save file I've posted below is usable for testing purposes, giving every elemental skill for the corresponding kill animations.
[sdds4game2.zip](https://github.com/Lime3DS/Lime3DS/files/14968844/sdds4game2.zip)

### Log File

N/A.

### System Configuration

CPU: AMD Ryzen 7 3700X
GPU/Driver: NVidia GeForce GTX 1650 4GB
RAM: DDR4 8G
OS: Windows 10 Home version 21H2

**Comment by Mortise976** [comment]
Are you using Vulkan ? Because the black box glitch was only fixed in OpenGL .

**Comment by leonmustdie** [comment]
Yep! That's probably my problem lol, I'll check to see if it works on OpenGL just in case.

**Comment by leonmustdie** [comment]
Bleh accidentally closed the issue. Didnt mean to lol

**Comment by leonmustdie** [comment]
Yep! It works with OpenGL, thanks Mortise! Didn't know Vulkan had issues with SMT4, but now I know :]

**Comment by Tayruu** [comment]
Setting to OpenGL fixes the kill animations, but it breaks the overlay effects in various locations - they have square holes in certain places. A 10% gap can be seen in Naruku and the overworld for example. This gap fills about 2/3rds of the screen if custom textures are loaded.

<img width="2048" height="615" alt="Image" src="https://github.com/user-attachments/assets/8c357d1c-465e-4976-897f-c9606218045b" />

I have attached the texture that causes this from the texturepack, but I don't think there's anything particularly wrong with the texture itself. (The black regions seem irrelevant given the shape of the hole in-game.)

<img width="2048" height="1024" alt="Image" src="https://github.com/user-attachments/assets/1e22931c-87cc-4250-8fe0-6473dcb6bb99" />
